translate/rus.json

```diff
@@ -162,7 +162,7 @@
 "exposure":"экспозиция",
 "extra options":"дополнительные настройки",
 "extra":"дополнительно",
-"fifty-move rule":"правило 50 ходов",
+"fifty-moves rule":"правило 50 ходов",
```

Comment:
It's not a typo, it's fifty moves, but a fifty-move rule.
It's correct.

Reply:
Then I am not sure if this is correct....
![Screenshot_20210303_003755_com android chrome](https://user-images.githubusercontent.com/77082125/109719037-1ff13600-7bb9-11eb-807c-68fdb726e6d7.jpg)

Reply:
Yeah, this one is then incorrect.

Reply:
Actually I just added code to the GUI that corrects the mistake, so we don't need to modify cutechess-cli.